### PR TITLE
6.1 metafiles

### PR DESCRIPTION
**Pull request by pedrobaeza**

### CODE REVIEW

**Comment by lmignon** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+[![Coverage Status](https://coveralls.io/repos/OCA/account-financial-reporting/badge.png?branch=6.1)](https://coveralls.io/r/OCA/account-financial-reporting?branch=6.1)
+
+OpenERP account financial reports
+=================================
+
+This project aims to deal with modules related to financial reports. You'll 
+find modules that print legal and official reports. This includes, among 
+others:
+
+* One module based on webkit and totally rewritten by camptocamp, for standard
```
Comment (rvalyi) [inline README.md:11]: camptocamp -> Camptocamp

**Comment by rvalyi** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (pedrobaeza) [inline README.md:11]: This is how it was originally written on Launchpad description. Should I modify it?

**Comment by laetitia-gangloff** [comment]
:+1:

**Comment by alejandrosantana** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (bwrsandman) [inline README.md:11]: Go for it. Also, replace OpenERP with Odoo in the title

Comment (pedrobaeza) [inline README.md:11]: Odoo is only for 8.0 branch. 7.0 and 6.1 must be kept with OpenERP, because all the references are for the old name.

Comment (bwrsandman) [inline README.md:11]: Noted

**Comment by yvaucher** [comment]
tests seams to give a false positive

 File "/home/travis/build/OCA/account-financial-reporting/account_financial_report_webkit/report/common_reports.py", line 313, in _get_st_fiscalyear_period. I guess this travis config is a big outdated

raise osv.except_osv(_('No period found'),'')

except_osv: ('No period found', '')

**Comment by pedrobaeza** [comment]
That's because a known bug in core. #5 is intended for workarounding the same on 7.0.

Regards.

File: README.md (hunk 1 above)
Comment (nbessi) [inline README.md:11]: @rvalyi is a real fan :tongue: and is correct it is Camptocamp.

**Comment by nbessi** [comment]
:+1: I'm in favor of #5 backport

**Comment by StefanRijnhart** [comment]
I'll merge this now, and leave backport of #5 to a future PR.

**Pull request merged**